server/resource_controller.go

```diff
@@ -290,6 +291,9 @@ func (u UnescapedJSONRenderer) Render(w http.ResponseWriter) (err error) {
 	data = bytes.Replace(data, []byte("\\u003e"), []byte(">"), -1)
 	data = bytes.Replace(data, []byte("\\u0026"), []byte("&"), -1)
 
+	// Convert "_id" to "id"
+	data = bytes.Replace(data, []byte("_id"), []byte("id"), -1)
```

Comment:
Of course, if you include those extra chars in the search, don't forget to put them in the replacement too!